packages/core/src/components/ui/Tooltip/Tooltip.js

```diff
@@ -62,6 +63,12 @@ const Tooltip = forwardRef(function Tooltip(props, ref) {
     [`${Classes.TOOLTIP}--persistent`]: persistent,
   })
 
+  if (_isEmpty(content)) {
+    return (
+      <>{children}</>
+    )
+  }
+
```

Comment:
this change is not required when we use following logic in Checkbox.js (mentioned in  comment above)

```
if(!_isEmpty(customhelp)) {
return content
}
```